test/standalone/Soil/Biogeochemistry/biogeochemistry_module.jl

```diff
@@ -73,7 +101,35 @@ end
     sources = ()
     boundary_conditions = (; top = (CO2 = top_bc,), bottom = (CO2 = bot_bc,))
 
-    soil_drivers = SoilDrivers(PrescribedMet(T_soil, θ_l), PrescribedSOC(Csom))
+    # Make a PrescribedAtmosphere - we only care about atmos_p though
+    precipitation_function = (t) -> FT(1)
```

Comment:
this is tricky, but we shouldnt write " (t) -> FT(1)" and pass that into the model. This is because we are asking the model to store this function:
```
function anonymous(t)
    return FT(1)
end
```
when this gets evaluated by the model each step, it will have to look for FT as a global parameter (in global scope), and this is slow. We *used* to get around this by writing `(t) ->eltype(t)(1)`, because `eltype(t)` can be computed without looking up anything in global scope. since we now arent guaranteed that the type of (t) will be the same FT as our simulation, we will just pass `(t) -> 1.0` and convert the output of this function internally to FT (where FT is accessible because it is stored in the model struct itself, or obtained from eltype(Y.soil.co2.C), or something like that.